sdk/servicebus/service-bus/src/serviceBusMessage.ts

```diff
@@ -501,7 +501,12 @@ export function fromRheaMessage(
     }
   }
 
-  const props: any = {};
+  type PartialWritable<T> = Partial<
+    {
+      -readonly [P in keyof T]: T[P];
```

Comment:
I'm not familiar with using a dash in front of readonly. What does that do?

Reply:
This is a nice way to do this, btw (Partial or not, I like that you added in typing and got rid of 'any'!)